### ISSUE TITLE
Figure out meaning of extra archive chunks

### ISSUE THREAD

**Issue by dertseha** [inkyblackness/ss-specs#14]
Save-game files contain 4 more chunks that are not included in archive.dat .
Try to figure out what they mean.

**Comment by dertseha** [comment]
Maybe one of them contains further auxiliary game state information. There are a few game settings that haven't been found yet for instance.

**Comment by jramstedt** [comment]
GAMEWRAP.C

// Save game schedule (resource #590)
// Save game schedule vec info (resource #591)

These seem to contain data for scheduled events.

**Comment by dertseha** [comment]
yup, with the source release, there's a whole bunch of revelations :)
first though I need to set various names straight.
